public/data/amehi0index.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Amelia hill",
```

Comment:
I think it'd be best if you capitalize the `h` here 😅 What do you think? Thanks.
```suggestion
  "name": "Amelia Hill",
```

Reply:
I think you are absolutely correct! I was sooo confused trying to figure everything out that I missed those. Thank you!